### ISSUE TITLE
Shift Styling to Tailwind CSS instead of bootstrap

### ISSUE THREAD

**Issue by mathanamathav** [mathanamathav/bunker-api#16]

**Comment by thatcatfromspace** [comment]
Is this issue still open for contribution?

**Comment by mathanamathav** [comment]
> Is this issue still open for contribution?

Yes , its open for contribution.

**Comment by thatcatfromspace** [comment]
Good to know! I'll create a pull request and work on the same.

**Comment by mathanamathav** [comment]
Absolutely! if you have some alternative design ideas or any specific preferences do try it out
ensure the changes align with the project's requirements. 
Looking forward to contributing!
